### DISCUSSION TITLE
high-scores/scores/881ccfb0-5f86-11ee-9d11-95ec66da2af4/

### DISCUSSION THREAD

**Discussion by giscus** [stb-gaming/high-scores#13] [Comments]
# high-scores/scores/881ccfb0-5f86-11ee-9d11-95ec66da2af4/

Sky Games High score

https://stb-gaming.github.io/high-scores/scores/881ccfb0-5f86-11ee-9d11-95ec66da2af4/

**Comment by tumble1999** [comment]
first automated submission, lets go
